src/procedures/procedures.11tydata.yaml

```diff
@@ -58,6 +58,34 @@ procedureOpeningCase:
           <li>Slightly lift off the enclosure on your left.</li>
           <li>Slide the enclosure off towards your right.</li>
         </ul>
+procedureOpeningCaseCM5:
+  title: Opening the case
+  steps:
+    - title: Protecting from ESD
+      image-apply-filter: true
+      image: step-2-esd.png
+      content: |
+        <p>Grounding protects the components from the static electricity that can be naturally present (electrostatic discharge, ESD).</p>
+        <ul>
+          <li class="warning">Always: Touch the working surface before touching electronics. </li>
+          <li>Ideally: Use ESD protective equipment, such as ESD table mats and grounding cords.</li>
+        </ul>
+    - title: Removing the case - bottom part
+      image: cm5-remove-thumb-screws.jpg
+      content: |
+        <ul>
+          <li>Remove the 4 screws on the bottom of the Home Assistant Yellow Kit.</li>
+          <li>Keep the thumb screws close by; you will need them again later.</li>
+        </ul>
+    - title: Removing the case - top part
+      image: cm5_remove_cover_480.webp
```

Comment:
Could we maybe standardize on naming here too (add step? Altough it has it's downside...) But maybe `_` vs `-` which is used everywhere else so far.

Reply:
no. adding the step number in the image file is not helpful. when the procedure changes or the image is reused elsewhere the step number is incorrect.